src/main/java/com/cinema_booking/booking/room/dto/RoomDTO.java

```diff
@@ -0,0 +1,21 @@
+package com.cinema_booking.booking.room.dto;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+import com.cinema_booking.booking.movie.model.Movie;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class RoomDTO {
+    private UUID id;
+    private String name;
+    private List<Movie> movies;
```

Comment:
So if we follow idea of creating different dtos for different purposes, we can then have a pojo like this:

`public class RoomDTO {
     private UUID id;
     private String name;
     private List<MovieResponse> movies;
     private Set<SeatResponse> seats;
}`

Would be much cleaner, and mapstruct will map nested objects for us